luzer/luzer.c

```diff
@@ -324,6 +317,133 @@ load_custom_mutator_lib(void) {
 	return 0;
 }
 
+/**
+ * Tries to asses how much bytecode there are loaded.
+ *
+ * I looked into lua's introspection capabilities, could not find anything good.
+ * There is https://github.com/leegao/see.lua but I don't think it is a good idea
+ * to make a PR with code for 5 different versions of a library to someone else's lib.
+ * Their idea is simple - decode bytecode in runtime.
+ * There is also https://github.com/siffiejoe/lua-getsize but reasoning is the same.
+ * They take signatures for 'struct Prototype' with them. Having them allow to
+ * see what Lua interpreter thinks sizes are.
+ * So here we sit, in quite a pickle, yearning for a lua-native crossplatform solution.
+ *
+ * Basically, this is stupid and straigtforward - table tree walk from '_G'.
+ * '_G' is Lua's special table for global stuff.
+ * 'string.dump' works even in latest LuaJIT. Bytecode is not crossplatform but we don't
+ * need that.
```

Comment:
I see a limitation of this approach: all Lua modules must be loaded before running the fuzzing process. Right?

Reply:
Right. In theory, we could update counters at runtime, but I would need to test if LF is okay with that. Tbh I guess second estimation strategy is better for the case when a lot of code is loaded dynamically. 

The much bigger limitation r/n as I see it is `local`.